core/src/main/scala/epollcat/unsafe/event.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Arman Bilge
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package epollcat.unsafe
+
+import scala.annotation.nowarn
+import scala.scalanative.posix.time._
+import scala.scalanative.unsafe._
+
+@extern
+@nowarn
+private[unsafe] object event {
+
```

Comment:
What is the provenance of these magic numbers?  
Are they macOS only or expected to work on any *BSD,
say FreeBSD?

Reply:
They are from here:
https://opensource.apple.com/source/xnu/xnu-7195.81.3/bsd/sys/event.h.auto.html

I do not know of their support beyond macOS. epollcat does not officially support FreeBSD or other BSDs. Lacking both a local machine and CI environment for it makes it difficult to support at this time. I'm happy to wait for a request.

Reply:
I suggest adding exactly that URL and a "Your mileage may vary on FreeBSD and others".
We have concurred before about "one thing done well" rather than "a skillion things done wrong".

I know the pain about doing primary development only on CI machines.  I give my SN PRs
on Windows two attempts, one to generate to smoke, and, if only a little, one to see if I can hack a fix"
Meanwhile, I utter words which, sad to admit, increase my karmic debt.

Reply:
Thanks for the chuckle :grin: I added the URL in https://github.com/armanbilge/epollcat/pull/27/commits/e24bd4a69bdd9c684feac613e4aae27d88136e5a.